apps/OboeTester/app/src/main/cpp/NativeAudioContext.h

```diff
@@ -347,7 +347,7 @@ class ActivityTestInput : public ActivityContext {
 
     oboe::Result startStreams() override {
         mInputAnalyzer.reset();
-        mInputAnalyzer.setup(getInputStream()->getFramesPerBurst(),
+        mInputAnalyzer.setup(std::max(getInputStream()->getFramesPerBurst(), callbackSize),
```

Comment:
We also have ActivityContext.getFramesPerBlock().
I wonder if that would also work.

Reply:
I don't like ActivityContext.getFramesPerBlock(). This does not differentiate the input and output stream.

Reply:
It seems to work.
We could also bounds check in FormatConverterBox. I am trying an assert.

Reply:
Right. I am looking at NativeAudioContext.cpp:244
It may not handle streams with different burst sizes properly when there is both INPUT and OUTPUT.